src/lib.rs

```diff
@@ -99,14 +99,17 @@ fn get_class(conn: &xcb::Connection, id: u32, options: &Options) -> Result<Strin
                     format!("{}", icon)
                 }
             }
-            None => format!("{}", class_display_name),
+            None => match options.general.get("default_icon") {
+                Some(default_icon) => format!("{} {}", default_icon, class_display_name),
```

Comment:
Here you should check if the option for running with `--no-names` is enabled

Reply:
Thanks for the review.

Does it still make sense to hide the name if we're using a default icon?
It would be tricky to determine which application it is in this case.

Reply:
Hmm, it might not, but knowing people we might end up in a situation like [this](https://xkcd.com/1172/). But I am inclined to think its unnecessary. I say we merge and then deal with whatever consequences might arrive :)